.github/workflows/pull-request.yml

```diff
@@ -22,15 +22,15 @@ jobs:
       matrix:
         chunk: [0, 1, 2, 3, 4, 5, 6, 7]
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3.0.2
```

Comment:
^ this and cache. why not v3?

Reply:
@mcspr Because 3.0.0 is an earlier version than 3.0.2?

Reply:
no? :)
action tags like v3 are updated during the action lifetime, they always point to the latest version available

Reply:
https://github.com/actions/cache/tags